src/constants/constants.ts

```diff
@@ -2,3 +2,4 @@ export const ADDRESSCHANGED = 'addressChanged'
 export const INCORPORATION_APPLICATION = 'incorporationApplication'
 export const ISFUTUREEFFECTIVE = 'isFutureEffective'
 export const ISIMMEDIATE = 'isImmediate'
+export const CORRECTION = 'correction'
```

Comment:
I think you should import Filing Types from another project instead.

This will also allow us to verify that header=="incorporationApplication" when we load the corrected filing (which you can add to onAppReady() in Correction.vue).

Reply:
Or at least separate the "loose constants" from the "filing type" constants :)

Reply:
Interested, do we have a Constants declaration file available like this? Are you thinking we relocate this to the bcrs-shared components and import? 
If that is the case, we should also do that with A LOT of our interfaces. That would be highly useful when if available across all the UI's :)

Reply:
We have some constants in Filings UI, as well as enums/types.

I wasn't thinking of moving this to Shared Components but it sounds like a good idea (except that it's an enum instead of a component) (and we'd need to import it using npm)...?

If someone has a good idea how to do this, please share. For now let's be Agile and not change it yet :)

Reply:
Let's do a little research on this, i can see value in that. Having interfaces/enums/constants shared across the 3 UI's would be amazing for consistency and cut down on the code.

Reply:
OK but if it takes significant time, please create a new ticket.

PS This feels similar to our common schemas...
https://github.com/bcgov/business-schemas

Reply:
Future...